env/elastica/composer.json

```diff
@@ -13,8 +13,6 @@
     ],
     "require": {
         "php": "^7.0",
-        "phpdocumentor/phpdocumentor": "^2.9",
```

Comment:
Can we please keep this in? I need this to generate the docs :-)

Reply:
Ok :) Probably I forgot to readd that one.